exercises/practice/triangle/.docs/instructions.append.md

```diff
@@ -0,0 +1,18 @@
+# About Enum
+The enum statement allows you to create a strongly typed set of labels. You can use that enumeration in the code without having to parse or check for spelling errors.
+
+Enumerations are internally represented as integers with a starting value of zero. By default, PowerShell assigns the first label in the list the value zero, and assigns the remaining labels with consecutive integers.
+
+Example:
+```
```

Comment:
You can make so this block will have syntax highlighting by writing `pwsh` after the "```"